addons/web_tour/static/src/scss/keyframes.scss

```diff
@@ -47,3 +47,36 @@
         transform: translate(0, 3px);
     }
 }
+
+$o-tip-border-width: 2px; // from tip.scss
+$o-tip-border-animated-width: 8px;
+$o-tip-border-width-correction: $o-tip-border-animated-width - $o-tip-border-width;
+$o-tip-border-color-light: #F9F9F9; // Odoo brand asset/Light gray
+$o-tip-border-color-dark: #8F8F8F; // Odoo brand asset/Gray
```

Comment:
Not per say... actual values appear in some elements, but when assigned to variables they look like brand-related info which might be overridden in a theme - which might lead to non-contrasting colors in some configurations.
I switched to white & gray because they are already used within the tip component.

Reply:
I meant using a variable from /web/primary_variables.scss which are not dependent on any theme, but indeed I don't think there is a good candidate there. Keep your hardcoded values :+1: